utils.py

```diff
@@ -214,6 +215,66 @@ def exp(self):
         D = ratio.numerator.coeffs[0]
 
         return tf(1, 1, deadtime=D)
+    
+    def bode(self, omega = numpy.logspace(-2, 2, 1000), plot = False, \
+             radians = False):
+        """
+        This function returns the bode response of the transfer function.
+        
+        Inputs:
+            omega:  A numpy array of frequencies over which to evaluate the
+                    transfer function.
+            plot:   State whether the magnitude and angle plots should be
+                    displayed.
+            radians: State whether the angle returned and angle plot should be
+                    in radians instead of degrees (default).
+        
+        Returns three arrays:
+            Omega:  The frequency range over which the transfer function was 
+                    evaluated.
+            G_mag:  The magnitudes of G at the relevent frequencies.
+            G_angle: The angles of G at the relevant frequencies.
+            
+        """
+        
+        # Convert tf object to a function.
+        def G(s):
```

Comment:
You don't need this function - it's already in `tf.__call__`, so you can just do `self(s)` where you're currently doing `G(s)`